v21.2/cluster-settings.md

```diff
@@ -16,7 +16,7 @@ In contrast to cluster-wide settings, node-level settings apply to a single node
 ## Settings
 
 {{site.data.alerts.callout_danger}}
-Many cluster settings are intended for tuning CockroachDB internals. Before changing these settings, we strongly encourage you to discuss your goals with Cockroach Labs; otherwise, you use them at your own risk.
+These cluster settings have a broad impact and affect all applications, workloads, and users running on a CockroachDB cluster. For some settings, a [session setting](set-vars.html#supported-variables) could be a more appropriate scope.
```

Comment:
To keep some of the original message:

`have a broad impact to CockroachDB internals and affect all...`

Reply:
Fixed.